samples/compiler_plugins/xnnpack/src/xnnpack/IR/XnnpackOps.td

```diff
@@ -40,4 +40,15 @@ def Xnnpack_BatchMatrixMultiplyOp : Xnnpack_Op<"batch_matrix_multiply", []> {
   }];
 }
 
+def Xnnpack_FullyConnectedNcQd8F32Qc4wOp : Xnnpack_Op<"fully_connected_nc_qd8_f32_qc4w", []> {
+  let summary = "Fully connected layer with int8 input, int4 weights, and f32 output";
+  let arguments = (ins AnyRankedTensor:$a,
```

Comment:
Add a TODO comment for adding other parameters such as bias and activation (min & max).